knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -78,7 +86,130 @@ func (r *ReconcileKnativeKafka) Reconcile(request reconcile.Request) (reconcile.
 		return reconcile.Result{}, err
 	}
 
-	// NOT IMPLEMENTED YET
+	// TODO: check for deletion
+	// if original.GetDeletionTimestamp() != nil {
+	//		return reconcile.Result{}, r.delete(original)
+	//	}
```

Comment:
Deletion not implemented yet. We need to delete stuff manually as we will have resources in other namespaces and those can't be garbage collected. Will implement this with a separate PR.

Reply:
that's fine w/ me  +1